### PR TITLE
Update all libraries

### PR DESCRIPTION
**Pull request by nre-ableton**
- Update to CodeNarc 1.6.1
- Update to GMetrics 1.1
- Update to SLF4J 1.7.30

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -7,7 +7,7 @@ FROM groovy:jre8
 
 USER root
 
-ENV CODENARC_VERSION=1.6
+ENV CODENARC_VERSION=1.6.1
```
Comment (ala-ableton) [inline Dockerfile:10]: There is a typo in e99aba01999620917ed2db2a3a50dc80ad8b70bc's commit message: CodeNarce -> CodeNarc

**Comment by nre-ableton** [comment]
@ala-ableton When I made this PR, the changelog contained all of those `[BREAKING]` changes, but I guess they edited the release description since then. Good thing we didn't merge this PR too quickly. 😄

**Comment by ala-ableton** [comment]
@nre-ableton https://github.com/CodeNarc/CodeNarc/blob/master/CHANGELOG.md does contain several occurrences of "[BREAKING CHANGE]", but that's on the `master` branch.

**Comment by nre-ableton** [comment]
> @nre-ableton https://github.com/CodeNarc/CodeNarc/blob/master/CHANGELOG.md does contain several occurrences of "[BREAKING CHANGE]", but that's on the `master` branch.

@ala-ableton Yeah, these were also in the draft changelog for the release, but that has been fixed by now.

**Pull request merged**